transport/src/main/java/io/netty/bootstrap/ServerBootstrap.java

```diff
@@ -219,14 +219,27 @@ public ServerBootstrap validate() {
         private final ChannelHandler childHandler;
         private final Entry<ChannelOption<?>, Object>[] childOptions;
         private final Entry<AttributeKey<?>, Object>[] childAttrs;
+        private final Runnable enableAutoReadTask;
 
         ServerBootstrapAcceptor(
-                EventLoopGroup childGroup, ChannelHandler childHandler,
+                final Channel channel, EventLoopGroup childGroup, ChannelHandler childHandler,
                 Entry<ChannelOption<?>, Object>[] childOptions, Entry<AttributeKey<?>, Object>[] childAttrs) {
             this.childGroup = childGroup;
             this.childHandler = childHandler;
             this.childOptions = childOptions;
             this.childAttrs = childAttrs;
+
+            // Task which is scheduled to re-enable auto-read.
+            // It's important to create this Runnable before we try to submit it as otherwise the URLClassLoader may
+            // not be able to load the class because of the file limit it already reached.
```

Comment:
@normanmaurer - agreed this won't hurt anything and I'm not against merging but I'm not convinced this addresses the original issue. I wonder if the general issue (no more FDs when attempting to load a class) is better addressed by application configuration updates (limit FDs accepted and/or connections made so the JVM has enough FDs to use for book keeping).